scripts/test_source_env.sh

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env bash
```

Comment:
Thanks for writing this!

Can you add this to the linux tests in `.github/actions/`?

Reply:
I'm not sure where that is? I wondered how tests worked. Guess I ran out of steam last night before finding the runner.

Do you mean add this to the mac/linux tests in the .circleci/config.yml?  If so, is there a way to run that locally to see if they work?

Reply:
Sorry yes I meant to say `.circleci/config.yml`.  Testing just on linux should be fine.